BranchSDK/BNCEventUtils.h

```diff
@@ -0,0 +1,23 @@
+//
+//  BNCEventUtils.h
+//  BranchSDK
+//
+//  Created by Nipun Singh on 1/31/23.
+//
+
+#import <Foundation/Foundation.h>
+#import "BranchEvent.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BNCEventUtils : NSObject
```

Comment:
Let's add a discussion on why this utility is necessary. It's not obvious that this is to maintain a strong reference to BranchEvent until the StoreKit calls come in.